src/network/networkd-link.c

```diff
@@ -2085,6 +2091,10 @@ int link_configure(Link *link) {
         if (link->iftype == ARPHRD_CAN)
                 return link_configure_can(link);
 
+        r = link_configure_mp_tcp(link);
+        if (r < 0)
+                return r;
```

Comment:
Ah, sorry, my bad. Please move `link_configure_mp_tcp()` before `link_configure_can()`. Otherwise, `link_check_ready()` does not update the state to "configured" for CAN devices.